packages/@sanity/components/src/validation/ValidationList.tsx

```diff
@@ -44,11 +44,12 @@ export default class ValidationList extends React.PureComponent<Props> {
       element.scrollIntoView({behavior: 'smooth', inline: 'center'})
       this.scrollTimeout = setTimeout(() => {
```

Comment:
Perhaps not related to this commit, but shouldn't any previous timeouts be cleared before scheduling a new one here?

Reply:
Fixed (and rebased into the same commit)